ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS_LEAFS.yml

```diff
@@ -60,6 +60,5 @@ l3leaf:
               delay_resp: 12
               follow_up: 13
               sync: 14
-        clock_identity_prefix: "22:33:44"
```

Comment:
why are we removing the test for a custom prefix?

Reply:
There's plenty of tests of custom prefixes on the other nodes in the test repo. By removing this we're effectively testing the fully auto-generated ClockID, which unfortunately wasn't tested earlier.

Reply:
ack. Thanks